scripts/development/m3_stack/README.md

```diff
@@ -17,7 +17,7 @@ The environment variables that let's you configure this setup are:
 
 ## Usage
 
-Use the `start_m3.sh` and `stop_m3.sh` scripts.
+Use the `start_m3.sh` and `stop_m3.sh` scripts. Requires successful run of `make m3dbnode` from project root first.
```

Comment:
@nbroyles Isn't this code in `start_m3.sh` takes care of running make, without running make on your own?
```
if [[ "$M3DBNODE_DEV_IMG" == "0" ]] || [[ "$FORCE_BUILD" == true ]] || [[ "$BUILD_M3DBNODE" == true ]]; then
    prepare_build_cmd "make m3dbnode-linux-amd64"
    echo "Building m3dbnode binary first"
    bash -c "$build_cmd"

    docker-compose -f docker-compose.yml up --build $DOCKER_ARGS m3db_seed
else
    docker-compose -f docker-compose.yml up $DOCKER_ARGS m3db_seed
fi
```